core/src/foundations/ionic.root.scss

```diff
@@ -0,0 +1,117 @@
+/**
+ * Ionic Design System
+ */
+
+:root {
+  --color-primary-10: #f7faff;
```

Comment:
Are we intentionally not prefixing these CSS variables to avoid conflicting with the existing `--ion` variables?

Reply:
Yes, I was not sure at this moment what we would want to have going forward and I thought not using that a bit more safe. 

But that can be easily changed on the tokens script. Maybe it should be --ionic-, as this only affects the Ionic Design Systems and themes? But this is a decision that should be made on the scope of the rest of the Design Doc, as it also impacts the variables on the scope of each component for the Ionic Theme.

Reply:
Just added a prefix to all generated variables 👍